### ISSUE TITLE
ERROR middleware/middleware.go:24 Key "user" does not exist

### ISSUE THREAD

**Issue by ttrinh24** [0xJacky/nginx-ui#483]
**Describe the bug**
Unable to access the cluster node Terminal from the main node.

**To Reproduce**
1. Go to main node
2. Click on 'Link Start' button at 'Environments' in 'Dashboard' page.
3. Go to cluster node
4. Click on 'Terminal' menu
5. See error

**Expected behavior**
Can exec cluster node terminal from main node. 

**Screenshots**
![image](https://github.com/user-attachments/assets/c825be56-0f45-44a6-9952-68b7d788debc)

**Info**
 - Server OS: Debian 12.6
 - Server Arch: linux 5.15.0-117-generic x86_64
 - Nginx UI Version: [v2.0.0-beta.29](https://github.com/0xJacky/nginx-ui/releases/tag/v2.0.0-beta.29)
 - Browser: Chrome

**Additional context**
- Error logs:
```
2024-08-02 10:41:25 ERROR middleware/middleware.go:24 Key "user" does not exist
goroutine 456 [running]:
github.com/0xJacky/Nginx-UI/router.InitRouter.Recovery.func2.1()
/home/runner/work/nginx-ui/nginx-ui/internal/middleware/middleware.go:23 +0x65
panic({0x33aca40?, 0xc001b6b500?})
/opt/hostedtoolcache/go/1.22.5/x64/src/runtime/panic.go:770 +0x132
github.com/gin-gonic/gin.(*Context).MustGet(...)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/context.go:284
github.com/0xJacky/Nginx-UI/api.CurrentUser(0x0?)
/home/runner/work/nginx-ui/nginx-ui/api/api.go:16 +0x8d
github.com/0xJacky/Nginx-UI/api/user.SecureSessionStatus(0xc001afe300)
/home/runner/work/nginx-ui/nginx-ui/api/user/otp.go:174 +0x1c
github.com/gin-gonic/gin.(*Context).Next(0xc001afe300)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/context.go:185 +0x2b
github.com/0xJacky/Nginx-UI/router.InitRouter.Proxy.func7(0xc001afe300)
/home/runner/work/nginx-ui/nginx-ui/internal/middleware/proxy.go:18 +0x165
github.com/gin-gonic/gin.(*Context).Next(0xc001afe300)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v
2024/08/02 10:41:25 http: panic serving 127.0.0.1:44642: interface conversion: string is not error: missing method Error
goroutine 456 [running]:
net/http.(*conn).serve.func1()
/opt/hostedtoolcache/go/1.22.5/x64/src/net/http/server.go:1903 +0xbe
panic({0x36803e0?, 0xc001c3fe30?})
/opt/hostedtoolcache/go/1.22.5/x64/src/runtime/panic.go:770 +0x132
github.com/0xJacky/Nginx-UI/router.InitRouter.Recovery.func2.1()
/home/runner/work/nginx-ui/nginx-ui/internal/middleware/middleware.go:27 +0x11c
panic({0x33aca40?, 0xc001b6b500?})
/opt/hostedtoolcache/go/1.22.5/x64/src/runtime/panic.go:770 +0x132
github.com/gin-gonic/gin.(*Context).MustGet(...)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/context.go:284
github.com/0xJacky/Nginx-UI/api.CurrentUser(0x0?)
/home/runner/work/nginx-ui/nginx-ui/api/api.go:16 +0x8d
github.com/0xJacky/Nginx-UI/api/user.SecureSessionStatus(0xc001afe300)
/home/runner/work/nginx-ui/nginx-ui/api/user/otp.go:174 +0x1c
github.com/gin-gonic/gin.(*Context).Next(0xc001afe300)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/context.go:185 +0x2b
github.com/0xJacky/Nginx-UI/router.InitRouter.Proxy.func7(0xc001afe300)
/home/runner/work/nginx-ui/nginx-ui/internal/middleware/proxy.go:18 +0x165
github.com/gin-gonic/gin.(*Context).Next(0xc001afe300)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/context.go:185 +0x2b
github.com/0xJacky/Nginx-UI/router.InitRouter.AuthRequired.func6(0xc001afe300)
/home/runner/work/nginx-ui/nginx-ui/internal/middleware/middleware.go:48 +0xcf
github.com/gin-gonic/gin.(*Context).Next(0xc001afe300)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/context.go:185 +0x2b
github.com/0xJacky/Nginx-UI/router.InitRouter.IPWhiteList.func4(0xc001afe300)
/home/runner/work/nginx-ui/nginx-ui/internal/middleware/ip_whitelist.go:14 +0x4d
github.com/gin-gonic/gin.(*Context).Next(0xc001afe300)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/context.go:185 +0x2b
github.com/0xJacky/Nginx-UI/router.InitRouter.Recovery.func2(0x10?)
/home/runner/work/nginx-ui/nginx-ui/internal/middleware/middleware.go:32 +0x45
github.com/gin-gonic/gin.(*Context).Next(...)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/context.go:185
github.com/gin-gonic/gin.LoggerWithConfig.func1(0xc001afe300)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/logger.go:249 +0xe5
github.com/gin-gonic/gin.(*Context).Next(...)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/context.go:185
github.com/gin-gonic/gin.(*Engine).handleHTTPRequest(0xc00105e000, 0xc001afe300)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/gin.go:633 +0x892
github.com/gin-gonic/gin.(*Engine).ServeHTTP(0xc00105e000, {0x4a2f008, 0xc001aa8000}, 0xc000dc6240)
/home/runner/go/pkg/mod/github.com/gin-gonic/gin@v1.10.0/gin.go:589 +0x1b2
net/http.serverHandler.ServeHTTP({0xc001c3fc80?}, {0x4a2f008?, 0xc001aa8000?}, 0x6?)
/opt/hostedtoolcache/go/1.22.5/x64/src/net/http/server.go:3142 +0x8e
net/http.(*conn).serve(0xc0017e2240, {0x4a38890, 0xc000e6d2f0})
/opt/hostedtoolcache/go/1.22.5/x64/src/net/http/server.go:2044 +0x5e8
created by net/http.(*Server).Serve in goroutine 1
/opt/hostedtoolcache/go/1.22.5/x64/src/net/http/server.go:3290 +0x4b4
```

**Comment by 0xJacky** [comment]
Thanks for your report, I will fix it as soon as possible.

**Comment by 0xJacky** [comment]
Fixed in v2.0.0-beta.30.
